android/src/fdroid/play/listings/it-IT/release-notes.txt

```diff
@@ -1,14 +1,13 @@
-• Fresh OpenStreetMap data as of July 10, 2023
-• Fixed several crashes
-• Support altitude, OsmAnd and Garmin colors in imported GPX tracks
-• Fixed pedestrian ETA for big slopes
-• Better car routing
-• Improved search ranking
-• Share button has been moved to the Place Page
-• Updated many translations
-• Added new shops and event venues types
-• Improved water tunnels and bridges rendering
-• More detailed railways
-• Added boundary=protected_area
+• New OpenStreetMap data as of August 14
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```

Comment:
• Nuovi dati di OpenStreetMap aggiornati al 14 agosto
• Aggiornati articoli di Wikipedia
• Aggiustata la ricerca per indirizzi in Cechia e Slovacchia
• Aggiunte linee isometriche mancanti per alcune regioni di Argentina, Cile, Kazakistan, Tibet, USA, Russia
• Rese linee isometriche più dense in alcune aree
• Risolto bug su Nexus 5X
• Aggiunte rotte dei traghetti alla mappa del mondo
• Ridisegnate icone dei parcheggi
• Ammodernato sistema di priorità per i luoghi d'interesse
• Tanti altri piccoli miglioramenti e bugfix
• Migliorata velocità di ricerca e indicazioni stradali